README.md

```diff
@@ -75,7 +75,7 @@ if (exists) {
 
 ### Logging a User In
 
-The following code is taken from [FHOauth.doOAuth()](app/src/main/java/com/feedhenry/oauth/oauth_android_app/FHOAuth.java#101).
+The following code is taken from `FHOAuth.java` (line 101).
```

Comment:
Let's avoid to add _line_  on this because it change from time to time and we always forget to change it on README ;)